docs/celery.rst

```diff
@@ -0,0 +1,119 @@
+======
+Celery
+======
+
+Official Celery documentation: http://docs.celeryproject.org/en/latest/
+
+What is it
+==========
+
+Celery is a library we use to perform tasks outside the bounds of an HTTP request.
+
+
+How to use celery
+=================
+
+All celery tasks should go into a tasks.py file or tasks module in a django app.
+This ensures that ``autodiscover_tasks`` can find the task and register it with the celery workers.
+
+These tasks should be decorated with one of the following:
+
+1. ``@task`` defines a task that is called manually (with ``task_function_name.delay`` in code)
+1. ``@periodic_task`` defines a task that is called at some interval (specified by ``crontab`` in the decorator)
```

Comment:
Should this be ```2.```?

Reply:
https://github.com/dimagi/commcare-hq/pull/23533/commits/756931a804fc2b309607b5e47e2d39c5752dfef0